src/tests/scripts/split_manpage/split_manpage_tests.py

```diff
@@ -196,10 +196,11 @@ def check_completeness_of_extracted_functions_and_macros(pmdk_path, lib_path):
     macros = get_macros(pmdk_path)
     functions_from_so_files = get_functions_from_so_files(lib_path)
     functions_windows_specific = get_functions_windows_only()
+    pmem2_async_functions_group_name = ['pmem2_async']
```

Comment:
why does it have to be a dict? If I understand correctly, there will never be any more items in it

Reply:
changed from dict to string

Reply:
👍